setup.cfg

```diff
@@ -1,4 +1,47 @@
-[egg_info]
-tag_build = .dev0
-#tag_svn_revision = yes
-#tag_date = yes
+[metadata]
+name = dill
+author = Mike McKerns
+maintainer = Mike McKerns
+license = BSD-3-Clause
+license_file = LICENSE
+description = serialize all of python
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://pypi.org/project/dill
+homepage = https://github.com/uqfoundation/dill
+download_url = https://github.com/uqfoundation/dill
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.5
+    Programming Language :: Python :: 2.6
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.1
+    Programming Language :: Python :: 3.2
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+    Topic :: Software Development
+platforms = Linux, Windows, Mac
+
+[options]
+packages = dill, dill.tools
+zip_safe = False
+python_requires = >=2.5, !=3.0.*
```

Comment:
`ctypes` requirement is missing.  It's needed for certain non-standard platforms, I believe like google cloud, if I remember correctly.

Reply:
isn't ctypes a part of python stdlih?

Reply:
Yes... or more precisely...  believe it or not... most of the time it is.   The STL contents are not all the same on all platforms and distributions, and last I checked, it was one of a few libraries that isn't always shipped as part of the STL.

Reply:
>and ctypes doesn't come installed by default with pypy or if using MacPorts.

I guess that in this case it should be installed using system package manager, like `apt` and `brew`, not `pip`.

Reply:
I can't control how people install `dill` -- the fact is sometimes the STL is slightly different due to how and where it was installed.

Reply:
Thue problem is not with dill, but with its native dependencies which are meant to be a part of python interpreter.

Reply:
I agree.  But, we have a practical problem here, about what to do about it.